config/crd/patches/webhook_in_policyservers.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: policyservers.policies.kubewarden.io
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          namespace: system
+          name: webhook-service
+          path: /convert
+      conversionReviewVersions:
+      - v1
```

Comment:
I don't remember anymore, does kubebuilder always generate this conversion webhooks?

Reply:
yes, that was generated by kubebuilder

Reply:
Perfect, thank you!